### ISSUE TITLE
[Luis] Overall - Deviations from the written script hard to follow

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-people-use-web-videos#166]
I know we're supposed to keep issues separate, but I wanted to call out an overall theme about the examples used in the video script. So I'm putting in one ticket for now to speak generally and presuming that individual issues can have separate tickets later.

The examples currently being proposed in the video script are hard for me to visualize and understand what we are telling people to gleam from them. Also, with many of them as they stand we would need detailed audio descriptions to describe the visuals because the script is not relaying the key points; in other scripts we've had to do this less.

Here's what I mean:

- Scene 2 & 3 - App for Club Basketball
  - I don't know many recreation teams that build & maintain their own apps with things like chat functions (and usable + accessible ones at that). This would be quite an undertaking if so - do we know this happens in real life?
  - Then I'm not sure what "records" Luis would keep. Are we saying he (and all the other team members) recall each of their stats and go home and record them? Is this realistic? 
  - Then, in terms of design, I can't picture from his dialogue what exactly is making these screens easy to use versus other apps. Seems like a calendar with "week numbers" listed, for instance, would start to get complex. Overall I'm not picturing what's making this calendar design (or other page designs) ideal (and how we'll show so many in a meaningful way in such a short time in the video).
  - In the written script the basketball reference is just about buying tickets to see games; not sure we should attempt including an app for the rec league.

- Scene 4 - Buying basketball tickets
  - This scene starts off saying there's a ticket purchasing website that's okay and then some that aren't. This makes me (as a viewer) want to see (and thus hear/audio describe) the example of the "good" website - whereas the written script doesn't try to relay that. I think this was added for balancing positive and negative experiences but it's seeming to add to the complexity instead.
  - Also, the descriptions of what makes the site difficult are hard to follow. Firstly, I wouldn't say "download a document" - that's a bit jargon-y and also doesn't explain why that's hard/jarring for Luis. Then, I'm not sure how "knowing where I am and how to get back" translates to a ticket buying system. What does a "good" vs. "bad" set-up include and does the visual layout matter? Lastly, what does it mean that he doesn't "understand what to put in the boxes"? The scene example is suggesting billing/shipping address and credit card but what about those would be harder on one site and not another (since we start with saying some ticketing sites are hard but not others)?
  - The written script has a different take on what makes a page better or worse and perhaps that's more of what we should incorporate??
  
- Scene 5 - Part-time job
  - Here I think the current written script could use a bit more detail about what an "office job" means. 
  - Nonetheless, firstly I wouldn't say "I'm not sure I can do it" - I would soften that as they've done in the written script.
  - Second, the statements about the difficulty with complex words and math seem to come out of nowhere, esp. since we don't know what kind of job Luis is looking for. 
  - Notably, saying "I use an online dictionary and calculator" doesn't feel like the language Luis would use - it seems more likely he would describe that he knows how to, say, "highlight a word with my mouse and have it show what it means".
  - Overall the written script more generally talks about being slowed down by having to look things up -- perhaps we can position the video a bit more like this. E.g., Luis would like to work full-time at an office job, but because things take him more time to do, it's hard to find someone who will hire him (and/or hard to feel confident he'll keep up). However, things wouldn't take so much time if there were less distractions (e.g., pop-ups) and more ways to easily understand Webpages (e.g., headings and easy layouts).

**Comment by nitedog** [comment]
Regarding apps for sport clubs, we use one for our wheelchair rugby team -- we use it to schedule practices and games, keep our player and game statistics, exchange messages and pictures, and tease the coach. A lot of the other aspects stem from the reference story.

Keeping this issue open for further perspectives.

**Comment by bakkenb** [comment]
Some of my reactions and thoughts...

Sequences 2 & 3:

- I do see that teams would use an existing calendaring, chatting, and team records app. They would log their team's wins and losses and keep up with their calendar of upcoming games. To me this is realistic.
- I do see Michele's point here. Maybe it needs to be strengthened why one app would be better than others. Right now it is about just knowing where you are in the app because of the use of team member pictures and other signposts that are prominent.

Sequence 4:

- Agree. Wondering why they would need to buy tickets to games they were playing in themselves. Not clear if this is for other basketball games. Other leagues or pro games. Maybe need a transition from Luis's club to buying tickets to other games.
- Agree with comments that there may need to be more explanation in Sequence 4 to "tell" what the issues are between a good and bad site.

Sequence 5:

- Agree with these comments. I also see a rewrite from COGA that we should consider in this area of what Luis's job is about.

**Comment by jade-mc** [comment]
I think some of these points might be better talked through, maybe Shadi and Brent you could talk through your experiences with how other filming has gone and the lead up to it (it's all still a bit abstract for me I suppose). I guess when I think about some of the filming, I lean towards being less specific about it (e.g. the job itself isn't that important, there might be a couple of generic office shots of Luis at his desk etc.) And there isn't necessarily time to go into a great amount of detail with the on screen examples so is it important to show this or go over it in the script more? 
And I didn't see any issues with the app, there are plenty of team/club/community management apps out there

**Comment by nitedog** [comment]
Incorporated the essence of this issue.
